src/client/pages/OfferNew/Checkout/InsuranceSummary.tsx

```diff
@@ -226,15 +227,19 @@ const getDetails = (
     ])
   }
 
+  const householdSize = getHouseholdSize(quoteDetails)
+
   detailsGroups.push([
     {
       key: 'antal-personer',
       label: textKeys.CHECKOUT_DETAILS_HOUSEHOLD_SIZE(),
       value:
-        getHouseholdSize(quoteDetails) === 1
+        householdSize === 0
+          ? '-'
+          : householdSize === 1
           ? textKeys.CHECKOUT_DETAILS_SINGLE_PERSON()
```

Comment:
i'm not a fan of nested ternaries as they imo get quite hard to read. Could we split this up into variables or named functions maybe?

Reply:
Sure, I can see that you would think that! 😀  I'll see what I can do!